### DISCUSSION TITLE
Middlewares

### DISCUSSION THREAD

**Discussion by kadiryazici** [frandiox/vite-ssr#71] [Show and tell]
Hi, I made middleware support for vite-ssr. I tried to make it look like Nuxt's middleware system.

The code and documentation can be found here: [vite-ssr-middleware](https://github.com/kadiryazici/vite-ssr-middleware).

If you have an idea or question you can type it to this thread 🙂.

**Comment by frandiox** [comment]
Thanks! @MarvinRudolph @johnRivs you might want to check this.

I've added a new section with community contributions to the readme 👍
